apps/web/src/lang/fr.json

```diff
@@ -7123,6 +7159,10 @@
     "defaultMessage": "L’étape suivante est la partie la plus importante de votre candidature. Il vous sera demandé de dire comment vous répondez aux exigences de compétences pour ce rôle.",
     "description": "Application step for skill requirements, introduction, description, paragraph one"
   },
+  "ilZlfH": {
+    "defaultMessage": "{count, plural, =0 {0 applicants au total} =1 {1 applicant au total} other {# applicants au total}}",
```

Comment:
Why is the word **total** needed here? It reads awkwardly and adds nothing to the context, seemingly.